hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -0,0 +1,25 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "pattgen"
+  import_testplans: ["hw/dv/tools/testplans/csr_testplan.hjson",
+                     "hw/dv/tools/testplans/intr_test_testplan.hjson",
+                     "hw/dv/tools/testplans/tl_device_access_types_testplan.hjson"]
+  entries: [
+    {
+      name: sanity
+      desc: '''
+            ...
+
+            Stimulus:
+              ...
+
+            Checking:
+              ...
+            '''
+      milestone: V2
+      tests: ["pattgen_sanity"]
+    }
+  ]
+}
```

Comment:
We probably need a bit more discussion around this test plan with the DV team. In order to not delay this PR, let's remove this file for now and add it back in a separate PR.

Reply:
I think the testplan is already merged in a separate PR that passed this one.  I'll just defer to what it says in there.

Reply:
@imphil the testplan is merged with PR #3446